### PR TITLE
add instructions about adding instructor as collaborator

### PR DESCRIPTION
**Pull request by amsnyder**

### CODE REVIEW

**Comment by AndyMcAliley** [comment]
We could add a link back to these instructions from [the end of issue_1](https://github.com/USGS-R/snakemake-introduction/blob/main/issues/issue_1.md#executing-the-pipeline):
```
See the instructions in [this repository's README](../README.md) for more details about the process of submitting PRs.
```

**Comment by amsnyder** [comment]
> We could add a link back to these instructions from [the end of issue_1](https://github.com/USGS-R/snakemake-introduction/blob/main/issues/issue_1.md#executing-the-pipeline):
> 
> ```
> See the instructions in [this repository's README](../README.md) for more details about the process of submitting PRs.
> ```

I like that! I've updated issue 1

**Comment by amsnyder** [comment]
I am just now noticing that this was raised in issue #25. Do you think this satisfies those instructions or are there any nuances that should be changed?

**Comment by AndyMcAliley** [comment]
This should take care of #25.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ The tutorial is organized as an ordered set of lessons, each of which introduces
 
 In order to walk through this tutorial, fork this repository into your own GitHub account. Then, clone your fork onto your local computer to work on.
 
-To begin the tutorial, open the markdown file [`issues/issue_0.md`](issues/issue_0.md) in GitHub. Issue 0 takes you through some basic setup and installation. You can then proceed through the following issues to continue through the tutorial.
+To begin the tutorial, open the markdown file `issues/issue_0.md` in GitHub. Issue 0 takes you through some basic setup and installation. You can then proceed through the following issues to continue through the tutorial.
```
[the remark this answers is not in the document]
Comment (amsnyder) [reply README.md]: This was a mistake! I think we merged in your suggested change after I made this pull request maybe? I just added it back

**Review by AndyMcAliley — APPROVED** [review]

**Pull request merged**